core/src/main/java/org/apache/servicecomb/core/Endpoint.java

```diff
@@ -61,6 +61,13 @@ public Object getAddress() {
     return address;
   }
 
+  public boolean isThirdPartyEndpoint() {
+    if (null == this.instance) {
```

Comment:
seems not so good.
we also need to support register 3rd services to SC, sometimes maybe also not allow send context header to that target.